src/codeyCommand.ts

```diff
@@ -10,23 +31,113 @@ export type SapphireMessageExecuteType = (
   client: SapphireClient<boolean>,
   // Message is for normal commands, ChatInputInteraction is for slash commands
   messageFromUser: Message | SapphireCommand.ChatInputInteraction,
-  initialMessageFromBot?: SapphireMessageRequest
+  initialMessageFromBot?: SapphireMessageRequest,
+  // Command arguments
+  args?: CodeyCommandArguments
 ) => Promise<SapphireMessageResponse>;
 
-// Can modify this as needed
-export type CodeyCommandOptions = {
-  name: string;
-  aliases: string[];
-  description: string;
-  detailedDescription: string;
-};
-
 // Type of response the Codey command will send
 export enum CodeyCommandResponseType {
   STRING,
   EMBED
 }
 
+// Command options
+export enum CodeyCommandOptionType {
```

Comment:
would u mind commenting these big things just so ppl in future can pick up slash commands easier?